update-netty-substitutions.sh

```diff
@@ -48,16 +48,43 @@ find "$DEST_RUNTIME" "$DEST_DEPLOYMENT" -type f -name "*.java" -exec sed -i '' \
     -e 's/io\.quarkus\.netty\.runtime\.graal/com.couchbase.quarkus.extension.runtime.nettyhandling.runtime.graal/g' \
     "{}" +
 
-# Replace copied NettyProcessor.java with our modified one
-echo "6 - Replacing NettyProcessor"
-cp -f ModifiedNettyProcessor.txt "$DEST_DEPLOYMENT"/NettyProcessor.java
-
 # Replace a line in the netty config which conflicts with existing netty extension
-echo "7 - Renaming Netty config item"
+echo "6 - Renaming Netty config item"
 sed -i '' 's/@ConfigRoot(name = "netty", phase = ConfigPhase.BUILD_TIME)/@ConfigRoot(name = "couchbase.netty", phase = ConfigPhase.BUILD_TIME)/' "$DEST_DEPLOYMENT/NettyBuildTimeConfig.java"
 
+#Delete two methods in NettyProcessor which we don't want to use
+echo "7 - Deleting code we don't want in NettyProcessor"
+
+sed -i '' '
+  /@BuildStep/{
+    N
+    /public RuntimeReinitializedClassBuildItem reinitScheduledFutureTask() /{
+      N
+      N
+      N
+      d
+    }
+  }
+' "$DEST_DEPLOYMENT/NettyProcessor.java"
```

Comment:
This is very flaky, but I also don't see a better way of doing it..  We'd need a CLI tool that can create a Java AST and prune it, to do this properly.
I think this will be good enough in practice.  It's very likely to result in a compile error if the methods change length, rather than e.g. a hard-to-debug runtime error.